### PR TITLE
Improvement: Class Highlight in Party Finder

### PR DESCRIPTION
**Pull request by Triggered0**
## What
Highlight selected class in Party Finder

<details>
<summary>Images</summary>

image
image

</details>

## Changelog Improvements
+ Added Class Highlighting for Party Finder. - Trîggered

### CODE REVIEW

**Comment by hannibal002** [comment]
Please expand the changelog wording. its unclear right now.

File: src/main/java/at/hannibal2/skyhanni/config/features/dungeon/PartyFinderConfig.java [hunk 1]
```diff
@@ -13,6 +15,43 @@ public class PartyFinderConfig {
...
+    public enum ClassEntry implements HasLegacyId {
+        NONE("None", 0),
+        TANK("Tank", 1),
+        HEALER("Healer", 2),
+        MAGE("Mage", 3),
+        ARCHER("Archer", 4),
+        BERSERK("Berserk", 5),
+        ;
+        private final String str;
+        private final int legacyId;
```
Comment (j10a1n15) [inline src/main/java/at/hannibal2/skyhanni/config/features/dungeon/PartyFinderConfig.java]: You don't need the legacyId in newly created enum config options

**Review by j10a1n15 — CHANGES_REQUESTED** [review]

**Comment by Triggered0** [comment]
> Please expand the changelog wording. its unclear right now.

is it ok rn?

**Comment by hannibal002** [comment]
I don't understand the feature. What is it for?
This feels redundant since we can already read and work with the currently selected role of the user.

**Comment by CalMWolfs** [comment]
> I don't understand the feature. What is it for? This feels redundant since we can already read and work with the currently selected role of the user.

Agreed

**Comment by CalMWolfs** [comment]
Closing this as you have not provided justification for this feature, feel free to reopen if you still want the feature

**Pull request closed without merging**